actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -191,6 +191,9 @@
     @Config("orbit.actors.defaultActorTTL")
     private long defaultActorTTL = TimeUnit.MINUTES.toMillis(10);
 
+    @Config("orbit.actors.localAddressCacheTTL")
+    private long localAddressCacheTTL = 0;
```

Comment:
yep, because cloud.orbit.actors.runtime.Hosting#actorDeactivated anyway removes it from the local addr cache and we don't want a race condition between Caffeine cache removing the local address entry and the deactivation system having to use the distributed cache to deactivate an actor.